prober/config.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/sethvargo/go-envconfig"
+)
+
+type config struct {
+	ProjectID              string        `env:"PROBER_PROJECT_ID,required"`
+	GCSBucketID            string        `env:"PROBER_BUCKET_ID,required"`
+	BigQueryDataSetID      string        `env:"PROBER_BIGQUERY_DATASET_ID,required"`
+	MappingTableID         string        `env:"PROBER_MAPPING_TABLE_ID,required"`
```

Comment:
I'd suggest implementing prober for mapping and policy in one PR so that we can figure out what the structure looks like

Reply:
I thought it's better to divide PR into smallest possible pieces so I didn't include the policy part. 
For probing policy service, only a new field `PolicyTableID` needs to be added.

Reply:
I doubt whether we should put mapping and policy prober into one main function/config.

Reply:
If we want one prober to probe both mapping and policy service, this is necessary.